aws/service_v2.go

```diff
@@ -957,6 +1007,17 @@ func ServerlessApplicationRepositoryClient(ctx context.Context, d *plugin.QueryD
 	return serverlessapplicationrepository.NewFromConfig(*cfg), nil
 }
 
+func ServiceQuotasRegionalClient(ctx context.Context, d *plugin.QueryData) (*servicequotas.Client, error) {
```

Comment:
Is there a non-regional equivalent? I'm curious as to why this isn't called `ServiceQuotasClient`.